docs/guide/personalization.md

```diff
@@ -106,7 +106,7 @@ ezrecommendation:
                     method: basic
                     login: 12345
                     password: 1234-5678-9012-3456-7890
-                document_root: '%kernel.root_dir%/../web/var/export/'
+                document_root: '%kernel.project_dir%/web/var/export/'
```

Comment:
If this should be exposed via http server then it should be rather `document_root: '%kernel.project_dir%/public/var/export/'`. @ciastektk @damianz5 correct me if I'm wrong.